app/assets/stylesheets/application.scss

```diff
@@ -11,3 +11,11 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+.notice {
+  margin: 0px;
+}
+
+.container {
```

Comment:
this container will affect all the containers in different pages?

Reply:
yes, but I checked it! It should normally work for all pages.